ui/src/app/pages/workspace/workspace-wrapper.tsx

```diff
@@ -28,6 +34,78 @@ import {
   runtimeStore,
   useStore,
 } from 'app/utils/stores';
+import { zendeskBaseUrl } from 'app/utils/zendesk';
+
+const styles = reactStyles({
+  newCtNotification: {
+    padding: 16,
+    marginTop: 16,
+    marginLeft: '2rem',
+    width: 'calc(100% - 5rem)',
+    boxSizing: 'border-box',
+    borderWidth: '1px',
+    borderStyle: 'solid',
+    borderRadius: '5px',
+    color: colors.primary,
+    fontFamily: 'Montserrat',
+    letterSpacing: 0,
+    lineHeight: '22px',
+    borderColor: colors.accent,
+    backgroundColor: colorWithWhiteness(colors.accent, 0.85),
+  },
+});
+
+interface NotificationProps {
+  isNewCT: boolean;
+  onCancel: () => void;
+}
+const MaybeNewCtNotification = (props: NotificationProps) => {
+  const dataPathsLink = (
+    <StyledExternalLink
+      href={`${zendeskBaseUrl()}/en-us/articles/4616869437204-Controlled-CDR-Directory`}
+    >
+      This resource
+    </StyledExternalLink>
+  );
+  const dataFilesLink = (
+    <StyledExternalLink
+      href={`${zendeskBaseUrl()}/en-us/articles/4614687617556-How-the-All-of-Us-Genomic-data-are-organized`}
+    >
+      this resource
+    </StyledExternalLink>
+  );
+  const cohortBuilderLink = (
+    <StyledExternalLink
+      href={`${zendeskBaseUrl()}/en-us/articles/360039585591-Selecting-participants-using-the-Cohort-Builder-tool`}
+    >
+      this resource
+    </StyledExternalLink>
+  );
+
+  return (
+    props.isNewCT && (
```

Comment:
since there is nothing to do in this class if the CT is not new, why do we have that decision here, why not in the parent class calling this class?

Reply:
ah - yes good point, thanks.  fixed.